src/components/pages/browse-programs-view/DashboardViewContainer.jsx

```diff
@@ -0,0 +1,88 @@
+import { Avatar, Button, Card, Col, Row } from 'antd';
```

Comment:
remove any unused imports

Reply:
also, the React version we're using in this project doesn't allow for us to omit `import React from "react";`, even in JSX files unfortunately. So go ahead and toss React as an import at the top as well.